src/MMM01.md

```diff
@@ -0,0 +1,403 @@
+The correct ROM header (with Nintendo logo) therefore needs to be located at offset `(size - 32 KiB) + $100` in the ROM rather than the usual `$0000 + $100` (which contains the header of the first game in the collection instead).
+MMM01 cartridges have a simple menu program in this last 32 KiB, which manipulates the additional MMM01 control bits, allowing game selection and setting the game size, before entering "mapped" mode and booting the selected game (see "Mapping Enable" below).
+
+As the last 32 KiB of the ROM are reserved for the cartridge menu, it's best to pack games into the ROM from largest to smallest to avoid having a game overlap the menu.
+For example, the Taito Variety Pack contains three 128 KiB games and one 64 KiB game in a 512 kiB ROM chip, leaving 32 KiB unused and 32 KiB for the menu.
+
+## Memory
+
+### 0000-3FFF - ROM Bank $X0 (Read Only)
+
+On startup (in "unmapped" mode), this is mapped to the first half of the menu program in the last 32 KiB of the ROM.
+
+When a game is mapped, this area normally contains the first 16 KiB (bank 00) of the game ROM.
+
+If [multiplex is enabled[(<#Multiplex Enable>)], entering mode 1 allows mapping game ROM banks $20, $40, and $60 to this region.
+
+#### Addressing diagrams
+
+When "unmapped":
+
+```
+                                    /------------- In a smaller cartridge, only the needed
+                                    |              bits are used (e.g 512 KiB uses 19 bits)
+                  /---------------------------\
+Bits: 22 21 20 19 18 17 16 15 14 13 12 .. 01 00
+      \_____________________/  | \____________/
+                 |             |      \----------- From Game Boy address
+                 |             \------------------ Always 0
+                 \-------------------------------- Always 1
+```
+
+Mapped, multiplex disabled:
+
+```
+Bits: 22 21 20 19 18 17 16 15 14 13 12 .. 01 00
+      \___/ \___/ \____________/ \____________/
+        |     |          |            \----------- From Game Boy address
+        |     |          \------------------------ bitwise and of ROM Bank Low & ROM Bank Mask
+        |     \----------------------------------- ROM Bank Mid
+        \----------------------------------------- ROM Bank High
+```
+
+### 4000-7FFF - ROM Bank $01-7F (Read Only)
+
+On startup (in "unmapped" mode), this is mapped to the second half of the menu program in the last 32 KiB of the ROM.
+
+When a game is mapped, this area may contain any of the further 16 KiB banks of the game ROM, except game banks $00, $20, $40, or $60.
+If one of those banks is selected, the low bit is forced to 1 and that bank is mapped instead ($01, $21, $41, or $61).
+
+i.e. in mapped mode, if `(ROM Bank Low) & ~(ROM Bank Mask)` is equal to $00 (indicating bank $00 within the game ROM), `(ROM Bank Low) | 1` is used instead.
```

Comment:
We don't use `~` as pseudocode in text. Please review: https://github.com/gbdev/pandocs/wiki/Document-Style

Reply:
There are in fact no bitwise operations in the pseudocode section at all, if I'm not blind.